python/interpret_community/explanation/explanation.py

```diff
@@ -715,12 +718,15 @@ def get_raw_explanation(self, feature_maps, raw_feature_names=None):
             in a single element list can be passed.
         :param raw_feature_names: list of raw feature names
         :type raw_feature_names: [str]
+        :param eval_data: Evaluation data.
+        :type eval_data: pd.DataFrame
```

Comment:
this can be a dataframe or numpy array - see one of the other docs.
I haven't seen a case where this is sparse data though for raw features.

Reply:
Updated to include both np.ndarray and pd.DataFrame. Thanks!